### PR TITLE
22 volunteer registration

### PR DESCRIPTION
**Pull request by kapucko**
- volunteer registration view
- new consts defined in ticket and ticket_type  - enables a possibility to reuse them in other submodules

### CODE REVIEW

File: konfera/views.py [hunk 1]
```diff
@@ -87,3 +89,22 @@ def cfp_form_view(request, event_slug):
...
+
+    if request.method == "POST":
+        form = VolunteerRegistrationForm(request.POST)
+
+        if form.is_valid():
+            new_ticket = form.save(commit=False)
+            new_ticket.status = REQUESTED
+            new_ticket.type = volunteer_ticket_type
+            new_ticket.save()
+            return redirect('event_details', event_slug)
```
Comment (ricco386) [inline konfera/views.py]: Please add blank line for better readability

**Comment by kapucko** [comment]
merged :)

File: konfera/models/ticket.py [hunk 2]
```diff
@@ -10,6 +10,11 @@
 CHECKEDIN = 'checked-in'
 CANCELLED = 'cancelled'
 
+REQUESTED = 'requested'
+REGISTERED = 'registered'
+CHECKEDIN = 'checked-in'
+CANCELLED = 'cancelled'
```
Comment (ricco386) [inline konfera/models/ticket.py]: It seems after the merge you have both your consts and also consts from master. When you started working on it they werent there but in the mean time they were added t o master so now you have them twice. Please remove duplicates.

**Comment by kapucko** [comment]
sorry I was distracted, should be OK now.

**Comment by ricco386** [comment]
@kapucko sure no problem, thanks for the PR.

**Pull request merged**